[2024-05-18T19:01:46.403+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T19:01:46.421+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2022-09-18T01:00:00+00:00 [queued]>
[2024-05-18T19:01:46.428+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2022-09-18T01:00:00+00:00 [queued]>
[2024-05-18T19:01:46.428+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-18T19:01:46.439+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hello_task> on 2022-09-18 01:00:00+00:00
[2024-05-18T19:01:46.444+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=332) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-18T19:01:46.446+0000] {standard_task_runner.py:63} INFO - Started process 334 to run task
[2024-05-18T19:01:46.446+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_loader', 'hello_task', 'scheduled__2022-09-18T01:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmp7qqlyxeg']
[2024-05-18T19:01:46.448+0000] {standard_task_runner.py:91} INFO - Job 54: Subtask hello_task
[2024-05-18T19:01:46.485+0000] {task_command.py:426} INFO - Running <TaskInstance: data_loader.hello_task scheduled__2022-09-18T01:00:00+00:00 [running]> on host b5cbc22da81e
[2024-05-18T19:01:46.548+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loader' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2022-09-18T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-09-18T01:00:00+00:00'
[2024-05-18T19:01:46.550+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T19:01:46.551+0000] {logging_mixin.py:188} INFO - 2022-09-18/
[2024-05-18T19:01:46.552+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-18T19:01:46.553+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T19:01:46.560+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_loader, task_id=hello_task, run_id=scheduled__2022-09-18T01:00:00+00:00, execution_date=20220918T010000, start_date=20240518T190146, end_date=20240518T190146
[2024-05-18T19:01:46.581+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-18T19:01:46.594+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T19:01:46.595+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
